Detectors/TRD/workflow/src/trd-calib-workflow.cxx

```diff
@@ -33,10 +37,23 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
 {
   auto enableRootInp = configcontext.options().get<bool>("enable-root-input");
+
   WorkflowSpec specs;
-  if (enableRootInp) {
-    specs.emplace_back(o2::trd::getTRDCalibReaderSpec());
+
+  if (configcontext.options().get<bool>("vDriftAndExB")) {
+    if (enableRootInp) {
+      specs.emplace_back(o2::trd::getTRDCalibReaderSpec());
+    }
+    specs.emplace_back(getTRDVdAndExBCalibSpec());
   }
-  specs.emplace_back(getTRDVdAndExBCalibSpec());
+
+  if (configcontext.options().get<bool>("noise")) {
+    if (enableRootInp) {
+      specs.emplace_back(o2::trd::getTRDDigitReaderSpec(false));
+      specs.emplace_back(o2::trd::getTRDNoiseCalibSpec());
+    }
+    // specs.emplace_back(o2::trd::getTRDNoiseCalibAggregatorSpec());
```

Comment:
Just checking if this a commenting fluke. Probably not, right?

Reply:
Sure, but this is also WIP and some things can still be found by @Archita-Dash who should add to this the actual calibration part ;)